### DISCUSSION TITLE
Where ist 7.0 code base in git?

### DISCUSSION THREAD

**Discussion by krawalli** [YetiForceCompany/YetiForceCRM#17500] [General]
I want to report a simple bug in the downloaded 7.01 code. But there seems to be no branch that matches that code base.
E.g, I'm missing the ***app/Mail/Account.php*** file.

Isn't a public code base, so that even non Yetiforce members can provide and share their code?

**Comment by krawalli** [comment]
Why is it not possible to answer that question or show a simple reaction?

**Comment by Marco31Bike** [comment]
YETIFORCE 7 seems to be proprietary software with open source components according to license... releasing option to upgrade 6.5 to 7 is a kindness of developer

**Reply by diderich** [reply]
You can download the code from here: https://yetiforce.com/en/test-download.html

**Comment by krawalli** [comment]
However, also those open source components aren't here in git for version 7.0+ - and thats what I asked. What is open source without improvements from a community?

@diderich thanks, but sure I know the download - I am looking for the code base here in git to share fixes, code and enhancements.
